public/summer-slam-2017/index.html

```diff
@@ -0,0 +1,539 @@
+
+            <p class="subtitle-paragraph margin-top-20 margin-bottom-50 clear-fix">
+              <span class="bold">Come join your fellow hackers for our inaugural summer event - the SUMMER SLAAAAAAAAAM!</a></p>
+            <p class="subtitle-paragraph margin-top-20 margin-bottom-50 clear-fix">
+              We're trying out something new this year - a summer event to complement our traditional Holiday Social, with the hopes we'll be able to make some new friends, reconnect with old friends, and in general spend more time with our local hacker community.</p>
+            <p class="subtitle-paragraph margin-top-20 margin-bottom-50 clear-fix">
+              Join us <strong>Sunday, August 6th, from 2PM to 7pm</strong>, at Bent Rail Brewery in Broad Ripple! We will have the <em>first</em> ever Indy Hackers' Summer Slam. Everyone is welcome! Pythonistas, Javanese, COBOLians, Scalanites, PHPers, Rubyists, .NET nuts, Perlsians -- the works. No offense if we've forgotten your language of choice (there are a lot of programming languages).
+            </p>
+            <p class="subtitle-paragraph margin-top-20 margin-bottom-50 clear-fix">Good news for the parents in the audience - this is a <strong>kid-friendly event</strong>, and we hope you'll come and bring the next generation of Indianapolis hackers with you. As always, this event is <em>completely</em> <strong>free</strong>.
+            </p>
+            <p class="subtitle-paragraph margin-top-20 margin-bottom-50 clear-fix">
+              If you're into hacking, we want to see you. Come by, drink some drinks, have some food, and have some fun in the sun with your fellow hackers.
+            </p>
+
+            <a href="https://www.eventbrite.com/e/indy-hackers-summer-slam-tickets-34754152606" class="button-black clear-fix">
+              <span class="click-here"></span>
+              Get Your Free Ticket
+            </a>
+
+          </li>
+          <!-- /Main -->
+
+
+          <!-- Features -->
+          <li>
+
+            <h2>What's Happening?</h2>
+
+            <p class="subtitle-paragraph">
+              FOODS. DRINKS. MUSIC.
+            </p>
+
+            <div class="layout-50 clear-fix">
+
+              <!-- Left column -->
+              <div class="layout-50-left clear-fix">
+
+                <!-- Features list -->
+                <ul class="no-list list-2 clear-fix">
+
+                  <!-- Item -->
+                  <li>
+                    <h3>Food</h3>
+                    <p class="icon-1 icon-1-6">
+                      Yet to be determined, but it'll be delicious. There will be vegetarian options.
+                    </p>
+                  </li>
+                  <!-- /Item -->
+
+                  <!-- Item -->
+                  <li>
+                    <h3>Drink</h3>
+                    <p class="icon-1 icon-1-6">
+                      Yet to be determined, but it'll be local. It will also probably be vegetarian. I assume.
```

Comment:
Add specific mention of non-alcoholic drinks here I think, a la:

Drink: Lemonade, Soda, Beer or Wine. We'll keep you hydrated and happy.